### PR TITLE
WP 31 serve a skeleton version of the app

### PR DESCRIPTION
**Pull request by mmcgahan**
This code update looks for a `?skeleton` query param on any request and returns a bare-bones (get it?) version of the app that will be rendered entirely on the client, which will be the entry point for an offline-first service worker (or if you just hate server rendering, here's your chance to beat the system!).

This still requires setting up the global configuration properties in the initial Redux state so that the client-side app can communicate with the API using server-provided oauth data.

### CODE REVIEW

**Comment by sadafie** [comment]
> bare-bones (get it?)

😂😂😂😂

**Comment by sadafie** [comment]
@mmcgahan I was a little disconnected last week, but is this just an investigatory branch or are we for sure going this route?

**Comment by mmcgahan** [comment]
@sadafie we can think about other methods of requesting the skeleton route, but the ability to serve it so that the SW can cache the skeleton is critical to having a 'good' offline/fast load experience. I'm pretty happy with the change for now, though.

File: renderers/server-render.jsx [hunk 1]
```diff
@@ -49,55 +61,53 @@ const DOCTYPE = '<!DOCTYPE html>';
...
-		result
+		return {
+			statusCode,
+			result
+		};
 	};
-}
 
 /**
  * Curry a Redux store and auth tokens to privde a function that can dispatch
```
Comment (sadafie) [inline renderers/server-render.jsx]: No longer a curry

**Pull request merged**